.ci.yaml

```diff
@@ -8,46 +8,124 @@
 enabled_branches:
   - master
 
+platform_properties:
+  linux:
+    properties:
+      caches: >-
+        [
+          {"name":"builder_linux_engine","path":"builder"}
+        ]
+      os: Linux
+  mac:
+    properties:
+      caches: >-
+        [
+          {"name":"flutter_cocoapods","path":"cocoapods"},
+          {"name":"osx_sdk","path":"old_osx_sdk"},
+          {"name":"builder_mac_engine","path":"builder"}
+        ]
+      os: Mac-10.15
+  windows:
+    properties:
+      caches: >-
+        [
+          {"name":"builder_win_engine","path":"builder"}
+        ]
+      timeout: "180"
+      os: Windows-10
+
 targets:
   - name: Linux Android AOT Engine
     builder: Linux Android AOT Engine
+    recipe: engine
+    properties:
+      add_recipes_cq: "true"
+      build_android_aot: "true"
+      android_sdk_license: \n24333f8a63b6825ea9c5514f83c2829b004d1fee
+      android_sdk_preview_license: \n84831b9409646a918e30573bab4c9c91346d8abd
+    timeout: 60
     scheduler: luci
 
   - name: Linux Android Debug Engine
     builder: Linux Android Debug Engine
+    recipe: engine
+    properties:
+      add_recipes_cq: "true"
+      build_android_debug: "true"
+      android_sdk_license: \n24333f8a63b6825ea9c5514f83c2829b004d1fee
```

Comment:
what's with the leading newline?

Reply:
Based on https://stackoverflow.com/a/38381577, it seems standard for the android sdk license to prefix with newline (this is just a copy and paste of the existing property)

Reply:
interesting